node/src/components/block_synchronizer/config.rs

```diff
@@ -21,6 +22,9 @@ pub struct Config {
     pub need_next_interval: TimeDiff,
     /// Time interval for recurring disconnection of dishonest peers.
     pub disconnect_dishonest_peers_interval: TimeDiff,
+    /// Time interval after which synchronization is considered stalled if no successful sync
+    /// activity happened.
+    pub progress_stall_threshold_interval: TimeDiff,
```

Comment:
I would just call this `stall_threshold_interval`.

Reply:
Or even `stall_limit` since it's an upper bound on how long we'll allow a stall.

Reply:
This is covered in a separate PR: https://github.com/casper-network/casper-node/pull/3842